cmd/geth/main.go

```diff
@@ -93,10 +93,12 @@ var (
 		utils.SyncModeFlag,
 		utils.ExitWhenSyncedFlag,
 		utils.GCModeFlag,
-		utils.LightServFlag,
-		utils.LightBandwidthInFlag,
-		utils.LightBandwidthOutFlag,
-		utils.LightPeersFlag,
+		utils.LightServeFlag,
+		utils.LightLegacyServFlag,
+		utils.LightIngressFlag,
+		utils.LightEgressFlag,
+		utils.LightMaxPeersFlag, // Deprecated in favor of light.maxpeers, remove in 2021
+		utils.LightLegacyPeersFlag,
```

Comment:
Should the `// Deprecated in favor of light.maxpeers` added here?

Reply:
Removed all the other instances, code started to look weird with all these deprecation repetitions.